Tests.Unit.Presentation.Web/Services/ReferenceServiceTest.cs

```diff
@@ -563,22 +605,30 @@ public void Cannot_BatchUpdateExternalReferences_If_Set_Master_Fails()
         }
 
         [Fact]
-        public void Cannot_BatchUpdateExternalReferences_If_Not_Allowed_To_Modify()
+        public void Cannot_UpdateExternalReferences_If_Not_Allowed_To_Modify()
```

Comment:
what about the test for an update by id, but where the id does not identify an existing object?=